provision.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
+set -e
+
+function wrong_args() {
+  echo >&2 "Wrong number of arguments! Usage:"
+  echo >&2 "  $(basename "$0")"
+  echo >&2 "  $(basename "$0") region-name country-code-list [osm-file-download-url]"
+  echo >&2 "Example:"
+  echo >&2 "  $(basename "$0") belgium BE http://download.geofabrik.de/europe/belgium-latest.osm.pbf"
+  exit 1
+}
+
+function usage() {
```

Comment:
This usage text is a bit heavy to digest. It offers three ways of getting the osm data to the app:
1) no osm data at all - using air distance. We describe it as the first and the simplest way at the same moment we warn the user about downsides of this approach. I would rather read about this possibility after going through the "better" options.

2) uploading local osm file. Requires at least three additional `oc` calls. It's very easy to make mistakes in the steps and the `oc rsync` command does not work for me (see the other comment below). If we want to offer uploading local osm file, let's do it via a file upload dialog in the app, otherwise the UX will suffer.

Description of the additional required steps should in my opinion rather go into documentation - together with a complete example of how to do it, step by step. As a part of usage of this script, it distracts user from understanding how to use the script alone.

3) this is the preferred way, if I understand it correctly. Why is it the last one?